### ISSUE TITLE
pip install faild due to pyyaml 6.0 build wheel failure

### ISSUE THREAD

**Issue by samuel-deal-tisseo** [kcl-lang/kcl-py#31]
## Bug Report

PyYAML 6.0 has some install issue with newer cython, corrected on version 6.0.1

Could you release a 0.7.6 version of package kclvm with PyYaml dependency bumped to 6.0.1 ?

See for reference:
- https://github.com/yaml/pyyaml/issues/601
- https://github.com/yaml/pyyaml/issues/724

This is almost a duplicate of https://github.com/kcl-lang/kcl-py/issues/22

### 1. Minimal reproduce step (Required)

` pip install kclv`

### 2. What did you expect to see? (Required)

install successfully

### 3. What did you see instead (Required)

```
❯ pip install kclvm
Collecting kclvm
  Downloading kclvm-0.7.5.tar.gz (310 kB)
     ---------------------------------------- 310.3/310.3 kB 3.2 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting PyYAML==6.0 (from kclvm)
  Downloading PyYAML-6.0.tar.gz (124 kB)
     ---------------------------------------- 125.0/125.0 kB 1.8 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [54 lines of output]
      running egg_info
      writing lib\PyYAML.egg-info\PKG-INFO
      writing dependency_links to lib\PyYAML.egg-info\dependency_links.txt
      writing top-level names to lib\PyYAML.egg-info\top_level.txt
      Traceback (most recent call last):
        File "D:\archi\misc\pyinfra\.venv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
          main()
        File "D:\archi\misc\pyinfra\.venv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "D:\archi\misc\pyinfra\.venv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
                 ^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\build_meta.py", line 325, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\build_meta.py", line 295, in _get_build_requires
          self.run_setup()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
          exec(code, locals())
        File "<string>", line 288, in <module>
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\__init__.py", line 103, in setup
          return distutils.core.setup(**attrs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 185, in setup
          return run_commands(dist)
                 ^^^^^^^^^^^^^^^^^^
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\core.py", line 201, in run_commands
          dist.run_commands()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\dist.py", line 963, in run_command
          super().run_command(command)
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\dist.py", line 988, in run_command
          cmd_obj.run()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\command\egg_info.py", line 321, in run
          self.find_sources()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\command\egg_info.py", line 329, in find_sources
          mm.run()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\command\egg_info.py", line 550, in run
          self.add_defaults()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\command\egg_info.py", line 588, in add_defaults
          sdist.add_defaults(self)
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\command\sdist.py", line 102, in add_defaults
          super().add_defaults()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\command\sdist.py", line 251, in add_defaults
          self._add_defaults_ext()
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\command\sdist.py", line 336, in _add_defaults_ext
          self.filelist.extend(build_ext.get_source_files())
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        File "<string>", line 204, in get_source_files
        File "C:\Users\sdeal\AppData\Local\Temp\pip-build-env-gn53wg3f\overlay\Lib\site-packages\setuptools\_distutils\cmd.py", line 107, in __getattr__
          raise AttributeError(attr)
      AttributeError: cython_sources
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

### 4. What is your KCL components version? (Required)

* python: 3.12
* kclvm: 0.7.5
* wheel: 0.42.0
* pip: 24.0
* virtualenv: 20.25.1

**Comment by Peefy** [comment]
I've released kclvm 0.7.6 with the pyyaml 0.6.1. Thank you!

**Comment by Peefy** [comment]
Closed by #32
